crawl-ref/source/dat/des/portals/bailey.des

```diff
@@ -509,32 +508,34 @@ xxxxxxxxxxx..<..xxxxxxxxxxx
 xxxxxxxxxxxc+++cxxxxxxxxxxx
 xxxxxxxxxxxc...cxxxxxxxxxxx
 xxxxxxxxcccc...ccccxxxxxxxx
-xxxxxxxccw.......wccxxxxxxx
-xxxcccccww.......wwcccccxxx
-xxxc<+1+ww.......ww+1+<cxxx
-xxxc.cccww.......wwccc.cxxx
-xxxc.+1+ww.......ww+1+.cxxx
-xxxc.cccww.......wwccc.cxxx
-xxxc.+1+ww.......ww+1+.cxxx
-xxxc.cccww.......wwccc.cxxx
-xxxc.+1+ww.......ww+1+.cxxx
-xxxc.cccww.......wwccc.cxxx
-xxxc^+1+ww.......ww+1+^cxxx
-xxxcccccww.......wwcccccxxx
-xxxxxxxccw.......wccxxxxxxx
+xxxxxxxxxxxc...cxxxxxxxxxxx
+xxxxxxxxcccc...ccccxxxxxxxx
+xxxxxxxccW.......Wccxxxxxxx
+xxxcccccWW.......WWcccccxxx
+xxxc1.1.WW.......WW.1.1cxxx
+xxxc.cccWW.......WWccc.cxxx
+xxxc1.1.WW.......WW.1.1cxxx
+xxxc.cccWW.......WWccc.cxxx
+xxxc1.1.WW.......WW.1.1cxxx
+xxxc.cccWW.......WWccc.cxxx
+xxxc1.1.WW.......WW.1.1cxxx
+xxxc.cccWW.......WWccc.cxxx
+xxxc1.1.WW.......WW.1.1cxxx
+xxxcccccWW.......WWcccccxxx
+xxxxxxxccW.......Wccxxxxxxx
```

Comment:
A bit worried about water cheesing the bridge now that the water is shallow, but I guess there are enough kobolds and the bridge is wide enough and monsters "step aside for ranged friends" some now so it might be ok. 

An alternate suggestion would be to add hallways leaving from the ambush spots to the room south of the bridge, so that once the "back up and re path" part of the AI is triggered the kobolds open the door (releasing the back up) and come down the bridge.

Reply:
After playing the map I'm still concerned about water cheese but it's just luring the orc warriors into shallow water, the initial attackers close faster than I could get flame wands. The hail of brigand darts makes that pretty unattractive though, so maybe its ok?